job_definitions/notify_suppliers_of_brief_withdrawals.yml

```diff
@@ -0,0 +1,54 @@
+{% set environments = ['preview', 'staging', 'production'] %}
+---
+{% for environment in environments %}
+- job:
+    name: "notify-suppliers-of-brief_withdrawals-{{ environment }}"
+    display-name: "Notify suppliers that their responses have been cancelled when a brief has been withdrawn - {{ environment }}"
+    project-type: freestyle
+    description: "Send email notifications to supplier users if they have a completed application to a brief that has been withdrawn."
+    parameters:
+      - string:
+          name: WITHDRAWN_DATE
+          default: null
+          description: "Specify a given date, send all emails for that date"
+      - string:
+          name: BRIEF_ID
+          default: null
+          description: "Specify a single brief, requires withdrawn_date"
+      - bool:
+          name: DRY_RUN
+          default: false
+          description: "List notifications that would be sent without sending the emails"
+    triggers:
+      - timed: "H 8 * * *"
+    publishers:
+      - trigger-parameterized-builds:
+          - project: notify-slack
+            condition: UNSTABLE_OR_WORSE
+            predefined-parameters: |
+              USERNAME=brief-withdrawals
+              JOB=Notify suppliers of brief withdrawals {{ environment }}
+              ICON=:question:
+              STAGE={{ environment }}
+              STATUS=FAILED
+              URL=<${BUILD_URL}consoleFull|#${BUILD_NUMBER}>
+              CHANNEL=#dm-release
+    builders:
+      - shell: |
+
+          FLAGS=""
+
+          if [ -n "$WITHDRAWN_DATE" ]; then
+            FLAGS="$FLAGS --withdrawn_date=$WITHDRAWN_DATE"
+          fi
+
+          if [ -n "$BRIEF_ID" ]; then
+            FLAGS="$FLAGS --brief_id=$BRIEF_ID"
+          fi
+
+          if [ "$DRY_RUN" = "true" ]; then
+            FLAGS="$FLAGS --dry-run"
+          fi
+
+          docker run --user $(id -u) digitalmarketplace/scripts scripts/notify-suppliers-of-brief-withdrawal.py '{{ environment }}' "$DM_DATA_API_TOKEN_{{ environment|upper }}" $NOTIFY_API_TOKEN 3e43a131-8399-4674-89d3-b4f80d2e07d4 $FLAGS
```

Comment:
Do we need the `--user $(id -u)` arg here? I thought this was only if we needed local write permissions?

Reply:
Maybe not but I still need to test this on jenkins, will see if it works without, have changed for now